docs/core/metrics.md

```diff
@@ -433,7 +433,7 @@ The `logMetrics` decorator of the metrics utility can be used when your Lambda h
 
```

Comment:
Minor: There are two more occurrences of the word `raised` at [L277](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/996dba61795b526cbfc0b209119b781b80e3c488/docs/core/metrics.md?plain=1#L277) & [L282](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/996dba61795b526cbfc0b209119b781b80e3c488/docs/core/metrics.md?plain=1#L277), would you mind updating them to `thrown`?